pallets/omnipool/src/tests/scenario_04.rs

```diff
@@ -0,0 +1,114 @@
+use super::*;
+
+/// Auto-generated test
```

Comment:
would be helpful to have some docs over how these tests were generated and what is the scenario about